File: mycollab-web/src/main/java/com/esofthead/mycollab/module/project/view/parameters/TimeTrackingScreenData.java
package com.esofthead.mycollab.module.project.view.parameters;

import com.esofthead.mycollab.module.project.domain.criteria.ItemTimeLoggingSearchCriteria;
import com.esofthead.mycollab.vaadin.mvp.ScreenData;

public class TimeTrackingScreenData {
	public static class Search extends
			ScreenData<ItemTimeLoggingSearchCriteria> {

		public Search(ItemTimeLoggingSearchCriteria criteria) {
			super(criteria);
		}
	}
}

File: mycollab-web/src/main/scala/com.esofthead.mycollab.module.project.view.parameters/TimeTrackingScreenData.scala
package com.esofthead.mycollab.module.project.view.parameters

import com.esofthead.mycollab.module.project.domain.criteria.ItemTimeLoggingSearchCriteria
import com.esofthead.mycollab.vaadin.mvp.ScreenData

/**
 * @author MyCollab Ltd.
 * @since 5.0.3
 */
object TimeTrackingScreenData {

  class Search(param: ItemTimeLoggingSearchCriteria) extends ScreenData[ItemTimeLoggingSearchCriteria](param) {}

}
